boards/posix/nrf52_bsim/time_machine.c

```diff
@@ -161,12 +161,12 @@ void tm_update_last_phy_sync_time(bs_time_t abs_time)
  * Advance the internal time values of this device
  * until the HW time reaches hw_time
```

Comment:
Nit: Please also rename the parameter in the comment.
You can also just name the input `t`

Reply:
Changed it to `t`.